vendor.conf

```diff
@@ -7,3 +7,9 @@ github.com/pkg/errors   v0.8.0-1-g839d9e9
 github.com/rancher/go-rancher-metadata  60977d6
 github.com/vishvananda/netlink  da7420c
 github.com/vishvananda/netns    8ba1072
+golang.org/x/sys a646d33e2ee3172a661fc09bca23bb4889a41bc8
+github.com/gorillalabs/go-powershell 8a975e1f46d292e02c8825f112090196bbc20ac1
+github.com/orangedeng/win-route-netsh f06f27f1fa5955d41f4d344e89283b008900a2cf
+github.com/mitchellh/mapstructure 06020f85339e21b2478f756a78e295255ffa4d6a
+github.com/juju/errors master
+github.com/orangedeng/go-winnat 699b70068ac94a0794a23951f1037fbac947d0af
```

Comment:
Can we move the dependencies under rancher? instead of orangedeng?

Reply:
@orangedeng we should move it to rancher. if you transfer it to me directly, i'll transfer it to rancher